assets/icons.json

```diff
@@ -1,121 +0,0 @@
-{
```

Comment:
This has all of the GCB block icons:
![block-icons](https://user-images.githubusercontent.com/4063887/99108762-6fc9d400-25ad-11eb-85ba-acc748757f62.png)

It worked fine as a `.json` file before, but now that we have a React-based editor, these are all moved to their own React component files.